tasks/broad/ImputationTasks.wdl

```diff
@@ -873,11 +873,12 @@ task FindSitesUniqueToFileTwoOnly {
 task SplitMultiSampleVcf {
  input {
     File multiSampleVcf
+    Int nSamples
 
     String bcftools_docker = "us.gcr.io/broad-gotc-prod/imputation-bcf-vcf:1.0.7-1.10.2-0.1.16-1669908889"
     Int cpu = 1
     Int memory_mb = 8000
-    Int disk_size_gb = ceil(3*size(multiSampleVcf, "GiB")) + 100
+    Int disk_size_gb = ceil(21*nSamples*size(multiSampleVcf, "GiB")/(nSamples+20)) + 100
```

Comment:
This line is a bit mysterious without any context. Maybe link to this PR with explanation on the formula for future readers?

Reply:
Or just a code comment with the formula (or a note that this is not random)